apps/nextjs/package.json

```diff
@@ -2,33 +2,40 @@
   "name": "@oakai/nextjs",
   "version": "1.0.0",
   "private": true,
+  "license": "MIT",
   "scripts": {
+    "aila": "tsx scripts/aila-cli.ts",
     "build": "next build",
+    "build-storybook": "dotenv -e ../../.env -- storybook build",
```

Comment:
I wonder if this highlights that `storybook:build` would be a better convention?

Reply:
Yes I think we want more consistency